config/application.rb

```diff
@@ -11,6 +11,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
```

Comment:
we're on Rails 5.2 now, shouldn't we update this sentence?

Reply:
Thanks